apps/prairielearn/src/ee/pages/studentCourseInstanceUpgrade/studentCourseInstanceUpgrade.html.ts

```diff
@@ -0,0 +1,32 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+export function StudentCourseInstanceUpgrade({ resLocals }: { resLocals: Record<string, any> }) {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../../../pages/partials/head') %>", {
+          ...resLocals,
+        })}
+      </head>
+      <body>
+        ${renderEjs(__filename, "<%- include('../../../pages/partials/navbar') %>", {
+          ...resLocals,
+          // This won't actually render anything on the page; it just has to be non-null.
+          navPage: 'upgrade',
+        })}
+        <main class="container mb-4 text-center">
+          <div class="d-flex flex-column justify-content-center">
+            <i class="fa-solid fa-lock fa-2xl"></i>
+            <h1>Upgrade required</h1>
```

Comment:
I wasn't totally sure what the user-facing language should be for all this plan grant stuff. I don't necessarily want to come out and say "payment required", so I went with "Upgrade required" and named the pages/routes/etc. accordingly. How do folks feel about this?

Reply:
As we discussed in person this language is fine for now and we can update it if needed when the actual page is implemented.